File: b3042fb/app/src/main/java/com/example/b3042fb/MainActivity.java
package com.example.b3042fb;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

import com.google.firebase.firestore.CollectionReference;
import com.google.firebase.firestore.FirebaseFirestore;

public class MainActivity extends AppCompatActivity {

    private EditText editTextTitle;
    private EditText editTextDescription;
    private EditText editTextPriority;
    private TextView textViewData;

    private FirebaseFirestore db = FirebaseFirestore.getInstance();
    private CollectionReference notebookRef = db.collection("Notebook");

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        editTextTitle = findViewById(R.id.edit_text_title);
        editTextDescription = findViewById(R.id.edit_text_description);
        editTextPriority = findViewById(R.id.edit_text_priority);
        textViewData = findViewById(R.id.text_view_data);
    }

    @Override
    protected void onStop() {
        super.onStop();
    }

    public void addNote(View view) {
        String title = editTextTitle.getText().toString();
        String description = editTextDescription.getText().toString();
        if (editTextPriority.length() == 0) {
            editTextPriority.setText("0");
        }

        int priority = Integer.parseInt(editTextPriority.getText().toString());

        Note note = new Note(title, description, priority);

        notebookRef.add(note);
    }

    public void loadNotes(View view) {

    }
}

File: l2172db/app/src/main/java/com/example/l2172db/MainActivity.kt
package com.example.l2172db

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView
import com.example.l2172db.db.MyDbManager

class MainActivity : AppCompatActivity() {

    var edTitle: EditText? = null
    var edContent: EditText? = null
    var tvTest: TextView? = null
    val myDbManager = MyDbManager(this)


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        edTitle = findViewById(R.id.edTitle)
        edContent = findViewById(R.id.edContent)
        tvTest = findViewById(R.id.tvTest)
        tvTest?.text = ""
        myDbManager.openDb()
        val dataList = myDbManager.readDbData()
        for( item in dataList){
            tvTest?.append(item)
            tvTest?.append("\n")
        }
    }

    fun onClickSave(view: View) {
        tvTest?.text = ""
        myDbManager.openDb()
        myDbManager.insertToDb(edTitle?.text.toString(), edContent?.text.toString())
        val dataList = myDbManager.readDbData()
        for( item in dataList){
            tvTest?.append(item)
            tvTest?.append("\n")
        }



    }

    override fun onDestroy() {
        super.onDestroy()
        myDbManager.closeDb()
    }
}